runelite-client/src/main/java/net/runelite/client/ui/overlay/Overlay.java

```diff
@@ -51,4 +56,26 @@ public String getName()
 	{
 		return this.getClass().getSimpleName();
 	}
+
+	boolean hasRightClickMenu()
```

Comment:
Remove these 2 methods, the rightClickMenuOptions arraylist is already exposed, so use that? I would maybe wrap it in object and make builder or something, idk